netbench-orchestrator/src/ssm_utils/netbench_driver/s2n_quic_driver_github.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{GithubRustSource, NetbenchDriverType};
+
+pub fn s2n_quic_server_driver() -> NetbenchDriverType {
```

Comment:
I'm not sure where these functions are used, but would instead of having all of the different files could you just generate this with a loop over `Vec<(driver_name, repo_name)>`? These all seem to have very similar structure.

Reply:
The github/crates ones might be similar but LocalSource is more complicated and will need to be a separate file. I feel the separate files helps with readability and feel its ok since we wont be touching these files alot.